launchpad/launchpad_reward.gno

```diff
@@ -215,12 +227,10 @@ func calculateDepositReward() {
 
 	for _, project := range projects {
 		// // loop with project
-		// println("projectIdx", projectIdx)
 		// println("project.id", project.id)
 
 		// // early return if not active
 		// println("project.startHeight\t", project.startHeight)
-		// println("now\t\t\t", now)
```

Comment:
remove unused lines. pls check the changes and remove them all.

Reply:
left over, cleaned up d52e746a8096c94641a42388c350c2b59bf4775a